README.md

```diff
@@ -39,6 +39,11 @@ rmats2sashimiplot is intended to be used in a Unix-based environment.
 
 ## Install
 
+rmats2sasmimiplot can be installed simply through pip:
+```
+pip install -e https://github.com/Xinglab/rmats2sashimiplot
```

Comment:
I tried running: `pip install -e https://github.com/Xinglab/rmats2sashimiplot` and got this error:
```
ERROR: https://github.com/Xinglab/rmats2sashimiplot is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with bzr+http, bzr+https, bzr+ssh, bzr+sftp, bzr+ftp, bzr+lp, bzr+file, git+http, git+https, git+ssh, git+git, git+file, hg+file, hg+http, hg+https, hg+ssh, hg+static-http, svn+ssh, svn+http, svn+https, svn+svn, svn+file).
```